css/style.css

```diff
@@ -0,0 +1,9 @@
+
+  #nav-bar {    
+    position: fixed; 
+    top: 0;     
+  }
+
```

Comment:
The style.css file should be directly inside the root directory, it does not need a parent folder.

Reply:
- [ ] It might be a good practice to organize the project structure by having meaningful folders that hold CSS files or even javascript, otherwise, if you have another point of view please consider explaining the `why` your suggestion should be followed by the student so it wont feel unfair 👍🏼